### PR TITLE
Template developed for CRC1382

### PR DESCRIPTION
**Pull request by LukasCBossert**
This is a template which has been created for a medicine-based CRC.

### CODE REVIEW

File: shared/sfb1382_gut-liver-axis/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* https://rdmorganiser.github.io/terms/options/precaution_SFB 1382
+* https://rdmorganiser.github.io/terms/options/sharing_publication_SFB 1382
+* https://rdmorganiser.github.io/terms/options/LTP_SFB 1382
+* https://rdmorganiser.github.io/terms/options/server_SFB 1382
+* https://rdmorganiser.github.io/terms/options/storage-during-project_SFB 1382
+* https://rdmorganiser.github.io/terms/options/formats_SFB 1382
+* https://rdmorganiser.github.io/terms/options/data_size-SFB 1382
+* https://rdmorganiser.github.io/terms/options/lab_notebook
+* https://rdmorganiser.github.io/terms/options/requirements_SFB 1382
+* https://rdmorganiser.github.io/terms/options/yes_no_not_yet
```
Comment (Zack-83) [inline shared/sfb1382_gut-liver-axis/README.md:28]: I would say that these are new option_**sets**_.
Eight of them seem to me very much application-specific. From one of the latest conversations I deduce they should correspondingly have a new prefix.
The other three are general enough, so they could be welcome with the `https://rdmorganiser.github.io/` prefix.

File: shared/sfb1382_gut-liver-axis/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+ 
+
+In the catalog following conditions are needed:
+
+* https://rdmorganis+er.github.io/terms/conditions/personal_data
```
Comment (Zack-83) [inline shared/sfb1382_gut-liver-axis/README.md:34]: `https://rdmorganis+er.github.io/` --> typo / Tippfehler (ein `+` zu viel)

File: shared/sfb1382_gut-liver-axis/SFB1382_options.xml [hunk 3]
```diff
@@ -0,0 +1,740 @@
...
+		<text lang="en">No</text>
+		<text lang="de">Nein</text>
+		<additional_input>False</additional_input>
+	</option>
+	<optionset dc:uri="https://rdmorganiser.github.io/terms/options/data_size-SFB 1382">
+		<uri_prefix>https://rdmorganiser.github.io/terms</uri_prefix>
+		<key>data_size-SFB 1382</key>
+		<dc:comment/>
+		<order>0</order>
+		<conditions/>
```
Comment (Zack-83) [inline shared/sfb1382_gut-liver-axis/SFB1382_options.xml:59]: I am worried that a whitespace in a key or in a path might produce problems. (The same applies to the following options.)

**Review by Zack-83 — CHANGES_REQUESTED** [review]
Gut gemacht, aber ich empfehle ein paar Punkte bevor dem _Merge_ zu revidieren

**Comment by LukasCBossert** [comment]
Thanks for the feedback. I will work on the mentioned topics and request for a new merge.

**Pull request closed without merging**